client/character.lua

```diff
@@ -153,12 +151,14 @@ local function randomPed()
     pcall(function() exports['illenium-appearance']:setPedAppearance(PlayerPedId(), ped) end)
     SetModelAsNoLongerNeeded(ped.model)
 end
-
 ---@param citizenId? string
 local function previewPed(citizenId)
-    if not citizenId then randomPed() return end
+    local cID = citizenId
+    local characters = lib.callback.await('qbx_core:server:getCharacters')
```

Comment:
We already use this callback. We shouldn't need to use it again. There is also no need to relocalize the citizenId